common/fw/firmware-version.h

```diff
@@ -4,6 +4,6 @@
 #pragma once
 
 #define D4XX_RECOMMENDED_FIRMWARE_VERSION "5.12.12.100"
-#define SR3XX_RECOMMENDED_FIRMWARE_VERSION "3.26.1.0"
+#define SR305_RECOMMENDED_FIRMWARE_VERSION "3.26.1.0"
```

Comment:
Cmake is broken now
-- https://librealsense.intel.com/Releases/SR300/FW/SR3XX_FW_Image-.bin
CMake Error at common/fw/CMakeLists.txt:55 (file):
  file DOWNLOAD HASH mismatch

    for file: [/home/user/git/librealsense/build/common/fw/SR3XX_FW_Image-.bin]
      expected hash: [55237dba5d7db20e7c218975375d05b4210e9460]
        actual hash: [da39a3ee5e6b4b0d3255bfef95601890afd80709]
             status: [22;"HTTP response code said error"]